internal/attestation/renderer/chainloop/v01.go

```diff
@@ -116,7 +116,10 @@ func outputChainloopMaterials(att *v1.Attestation, onlyOutput bool) []*Provenanc
 		mdef := materials[mdefName]
 
 		artifactType := mdef.MaterialType
-		nMaterial := mdef.NormalizedOutput()
+		nMaterial, err := mdef.NormalizedOutput()
+		if err != nil {
```

Comment:
why do we continue on error, can you add a comment here?

Reply:
because v01 is actually deprecated and in fact we should probably remove it. Just to not to add more code.

Reply:
I should have added a comment on the reasoning, sorry.